ebos/eclgenericproblem_impl.hh

```diff
@@ -169,6 +173,24 @@ readRockParameters_(const std::vector<Scalar>& cellCenterDepths)
         const auto& num = eclState_.fieldProps().get_int(rock_config.rocknum_property());
         for (size_t elemIdx = 0; elemIdx < numElem; ++ elemIdx) {
             rockTableIdx_[elemIdx] = num[elemIdx] - 1;
+            auto fmtError =
+                [&num,elemIdx,&ijkIndex,&rock_config](const char* type, std::size_t size)
+                {
+                    return fmt::format("{} table index {} for elem {} read from {}"
+                                      " is larger than number of tables {}",
```

Comment:
Maybe we could say

> is out of bounds for the number of tables

instead of "is larger than"?

Reply:
done and done.